client/app/expenses/components/ItemDisplayPerMonth.tsx

```diff
@@ -0,0 +1,38 @@
+import { useRouter } from 'next/navigation'
+import { FiEdit } from 'react-icons/fi'
+import { Button } from '@/_components/ui'
+import { DeleteExpense } from './DeleteExpense'
+import { Expense } from '../types'
+
+type ItemDisplayPerMonthProps = {
+  item: Expense
+  userId: number
+}
+
+const ItemDisplayPerMonth = ({ item, userId }: ItemDisplayPerMonthProps) => {
+  const router = useRouter()
+  return (
+    <div className='flex flex-col gap-2'>
+      <div className='flex items-center justify-between gap-2 text-base font-semibold md:text-lg'>
+        <p className='overflow-auto truncate'>{item.itemName}</p>
+        <span>${item.fee}</span>
+      </div>
+      <div className='flex items-center justify-end gap-2'>
+        {userId === item.buyerId && (
+          <>
+            <Button
+              size={'smIcon'}
+              variant={'outline'}
+              onClick={() => router.push(`/expenses/${item.id}/edit`)}
+            >
+              <FiEdit />
+            </Button>
```

Comment:
Could you add an aria-label? We usually need to add an aria-label for the icon button!
I think I forgot some areas. I will check later...

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/119/commits/abb4b59112b2e29fd78a26b63ffb0433a18ddeb6